Submodels/MTSOL_call.py

```
@@ -238,25 +352,43 @@ def SetOperConditions(self,
         # Set momentum/entropy conservation Smom flag
         self.StdinWrite("S 4")
 
+        # Wait for completion of processing operating condition change
+        self.WaitForCompletion(completion_type=CompletionType.PARAM_CHANGE)
+
         # Exit the modify solution parameters menu
         self.StdinWrite("")
     
 
     def ToggleViscous(self,
+                      surface_IDs: list[int] = [3, 4]
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)